### ISSUE TITLE
slacker-asyncio it's probably no longer mantained

### ISSUE THREAD

**Issue by anxodio** [opsdroid/connector-slack#11]
slacker-asyncio it's the main dependency and seems abandoned.

The last commit is from May'16.
See this issue: https://github.com/gfreezy/slacker-asyncio/issues/7

**Comment by jacobtomlinson** [comment]
Yeah I was a little unsure of that dependancy when I started this connector as there had been no activity for two months even then.

Happy to explore alternatives or use the REST API directly.
